charts/rollups-validator-node/values.yaml

```diff
@@ -124,6 +124,87 @@ validator:
   ##
   extraVolumes: []
 
+  localnode:
+    enabled: false
+    storage:
+      blockchainData:
+        storageClass: standard
+        size: 2Gi
+      deployments:
+        storageClass: standard
+        size: 1Gi
+      machineSnapshots:
+        storageClass: standard
+        size: 2Gi
+    service:
+      type: ClusterIP
+      port: 8545
+
+    anvil:
+      enabled: false
+      image:
+        registry:
+        repository: sunodo/anvil
+        tag:
```

Comment:
I think we need a versioned tag here.

Reply:
We need to have a default value, and one can redefine it if it's needed.

It's better to use safe default for predictability.

Reply:
Got it !

Reply:
Fixed !